### ISSUE TITLE
Ability to turn off "joe is online, click to chat with him"

### ISSUE THREAD

**Issue by greggawatt** [cryptocat/cryptocat#3]
I don't think I need notifications on this, it would be nice to dial it down.

**Comment by kaepora** [comment]
Just so I understand, you want to be able to turn off some notifications without turning off all of them, correct? Because you *can* already [turn off notifications](https://crypto.cat/help.html#notify).

**Comment by greggawatt** [comment]
Yeah some, not all. I want to know when someone messages me but not when they are online

**Comment by kaepora** [comment]
Okay, sure. I'll add this in the near future.
